TTHAnalysis/cfg/crab/resubmitMultipleCrabJobs.sh

```diff
@@ -0,0 +1,51 @@
+export DATA=Fall17
```

Comment:
I think this file was not supposed to be included in the PR

Reply:
It wasn't. Removed it from this PR.